codingTest_jye/문자열-내림차순-정리.js

```diff
@@ -0,0 +1,17 @@
+//핵심은 sort()의 사용
+
+function solution(s) {
+    let answer = '';
+    let key = [];
+    
+    key = s.split('');
```

Comment:
```suggestion
    const key = s.split('');
```
이렇게 하는게 더 좋을 것 같아요. 
https://github.com/modern-agile-team/7term-assignment/pull/9/files#r1415044790
이 피드백 참조 해주세요.